Rego/SharepointConfig.rego

```diff
@@ -17,49 +27,68 @@ import data.report.utils.ReportDetailsString
 # SharingCapability == 1 New and Existing Guests
 # SharingCapability == 2 Anyone
 
-tests[{
-    "PolicyId" : "MS.SHAREPOINT.1.1v1",
-    "Criticality" : "Should",
-    "Commandlet" : ["Get-SPOTenant", "Get-PnPTenant"],
-    "ActualValue" : [Policy.SharingCapability],
-    "ReportDetails" : ReportDetailsBoolean(Status),
-    "RequirementMet" : Status
-}] {
-    Policy := input.SPO_tenant[_]
-    Conditions := [Policy.SharingCapability == 0, Policy.SharingCapability == 3]
-    Status := count([Condition | Condition = Conditions[_]; Condition == true]) == 1
+# If SharingCapability is set to Only People In Organization
+# OR Existing Guests, the policy should pass.
+tests contains {
+    "PolicyId": "MS.SHAREPOINT.1.1v1",
+    "Criticality": "Should",
+    "Commandlet": ["Get-SPOTenant", "Get-PnPTenant"],
+    "ActualValue": [SharingCapability],
+    "ReportDetails": ReportDetailsBoolean(Status),
+    "RequirementMet": Status
+} if {
+    some TenantPolicy in input.SPO_tenant
+    SharingCapability := TenantPolicy.SharingCapability
+    Conditions := [
+        SharingCapability == 0,
+        SharingCapability == 3
+    ]
+    Status := count(FilterArray(Conditions, true)) == 1
 }
+
 #--
 
 #
 # MS.SHAREPOINT.1.2v1
 #--
 
-tests[{
-    "PolicyId" : "MS.SHAREPOINT.1.2v1",
-    "Criticality" : "Should",
-    "Commandlet" : ["Get-SPOTenant", "Get-PnPTenant"],
-    "ActualValue" : [Policy.OneDriveSharingCapability],
-    "ReportDetails" : ReportDetailsBoolean(Status),
-    "RequirementMet" : Status
-}] {
-    Policy := input.SPO_tenant[_]
+# OneDriveSharingCapability == 0 Only People In Organization
+# OneDriveSharingCapability == 3 Existing Guests
+# OneDriveSharingCapability == 1 New and Existing Guests
+# OneDriveSharingCapability == 2 Anyone
```

Comment:
Just an observation.  Code would be more readable if we assigned these and used variables below. Now I have to go back infile to decode.

Reply:
Are you thinking along the lines of turning the values into constants?
![image](https://github.com/cisagov/ScubaGear/assets/47129205/1c5353d1-6f52-436f-9597-78f459e5a41e)
![image](https://github.com/cisagov/ScubaGear/assets/47129205/41334ae7-6682-441b-9d09-2edcd7c8ed16)

Reply:
yes